CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## next
 
+*Enhancements*
+- Uses `TASK_ID` environment variable as the `deployment_id` when rendering resource templates for better Shipit integration.
```

Comment:
Can you make `Shipit` a link to the github repo https://github.com/Shopify/shipit

Reply:
Done :)